estimagic/optimization/kernel_transformations.py

```diff
@@ -1,8 +1,34 @@
+"""Functions and derivatives thereof to transform external and internal params.
+
+Remarks on the mathematical notation:
+-------------------------------------
+
+We let :math:`X` denote the Cholesky factor of some covariance matrix :math:`S`.
+That is :math:`X X^\top = S`. We write :math:`\text{vec}(A)` for the column-wise
+vectorization of the matrix :math:`A` and we write :math:`\text{vech}(A)` for
+the row-wise half vectorization of :math:`A`. We denote the elimination
+matrix by :math:`L`, which fulfills :math:`L \text{vec}(A) = \text{vech}(A)`.
+For lower-triangular matrices :math:`A` we define the "lower-triangular"
+duplication matrix :math:`D`, which is not to be confused with the standard
+duplication matrix, and fulfills :math:`D \text{vech}(A) = \text{vec}(A)`. At
+last we define the so called commutation matrix :math:`K` which is given by the
+property that :math:`K \text{vec}(A) = \text{vec}(A^\top)`.
+
+Remarks on reference literature:
+--------------------------------
+
+The solutions on how to compute the jacobians implemented here can be found
+using matrix calculus. See for example 'Matrix Differential Calculus with
+Applications in Statistics and Econometrics' by Magnus and Neudecker. In
+specific cases we refer to posts on math.stackexchange.com.
+
+"""
```

Comment:
Replace all tinyurl references with the following ref. see https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google for an example.

```suggestion
.. rubric:: References

.. _post_mathoverflow:
   https://google.github.io/styleguide/pyguide.html

"""
```